lte/gateway/c/oai/tasks/mme_app/mme_config.c

```diff
@@ -640,14 +640,11 @@ int mme_config_parse_file(mme_config_t* config_pP) {
         config_setting_get_member(setting_mme, MME_CONFIG_STRING_TAI_LIST);
     if (setting != NULL) {
       num = config_setting_length(setting);
-      OAILOG_INFO(LOG_MME_APP, "Number of TAIs configured: %d\n", num);
-      AssertFatal(
-          num >= MIN_TAI_SUPPORTED,
-          "Not even one TAI is configured, configure minimum one TAI\n");
-      AssertFatal(
-          num <= MAX_TAI_SUPPORTED,
-          "Too many TAIs configured: %d (Maximum supported: %d)", num,
-          MAX_TAI_SUPPORTED);
+      if (num <= MIN_TAI_SUPPORTED) {
+        fprintf(
+            stderr,
+            "Not even one TAI is configured, configure minimum one TAI\n");
```

Comment:
Maybe reword as "No TAI is configured. At least one TAI must be configured.\n"

Reply:
Fixed in latest version